src/main/java/stelnet/market/MarketQueryBoard.java

```diff
@@ -56,10 +56,8 @@ public void createIntelInfo(TooltipMakerAPI info, ListInfoMode mode) {
     @Override
     public void createLargeDescription(CustomPanelAPI panel, float width, float height) {
         Size size = new Size(width, height);
-        GridRenderer renderer = new GridRenderer(size);
-        MarketData data = new MarketData(panel, queries);
-        renderer.setTopLeft(data.getTopLeft(size));
-        renderer.render(panel);
+        (new ControlRow(queries)).render(panel, size);
+        (new Queries(queries)).render(panel, size);
```

Comment:
I think this works without extra parentheses as well. It returns instance from the constructor and then we call the method on that instance.

Extra parentheses usually come into play when we want to cast between types.

```
        new ControlRow(queries).render(panel, size);
        new Queries(queries).render(panel, size);
```